### ISSUE TITLE
Support for Android TV

### ISSUE THREAD

**Issue by aecarvalho** [BeamlakAschalew/cinemax#22]
### Steps to reproduce

1. Installing Cinemax in Android TV firstly provides no shortcut for the app, even through TV App Repo. It can only be opened via settings>system apps>Cinemax>open 
2. Using Cinemax in Android TV is difficult as the buttons cannot be clicked using Android TV remote, using mouse is the only feasible option 

### Expected behavior

The app should work fine on Android TV just as it works on the smartphone, and provide same functionality

### Actual behavior

1) No application icon shown in Google TV home screen after installing app
2) Buttons cannot be clicked using Android TV remote

### Android version

Android 9

### Other details

Model Mi TV 4A
Arm v7 preferable

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by BeamlakAschalew** [comment]
Ok i will fix this ✌🏼

**Comment by DarkCrypt** [comment]
I know you're working on tv support, just letting you know same issues above exist with v1.4.0 on Android Tv 10.

No shortcut, app opens from settings->system apps but cannot get passed the login screen. The entire screen doesn't show properly and cuts off some of the 3 button options which cannot be clicked with remote.

You can reproduce this issue on phone by rotating to landscape. When I do this all 3 button options are completely off screen. Portrait is the only way to get passed this. Landscape works properly on all other screens.

**Comment by BeamlakAschalew** [comment]
> I know you're working on tv support, just letting you know same issues above exist with v1.4.0 on Android Tv 10.
> 
> No shortcut, app opens from settings->system apps but cannot get passed the login screen. The entire screen doesn't show properly and cuts off some of the 3 button options which cannot be clicked with remote.
> 
> You can reproduce this issue on phone by rotating to landscape. When I do this all 3 button options are completely off screen. Portrait is the only way to get passed this. Landscape works properly on all other screens.

I forgot to support Android TV, I'll put that in my trello right now. Are the 3 buttons the navigation buttons?

**Comment by BeamlakAschalew** [comment]
> I know you're working on tv support, just letting you know same issues above exist with v1.4.0 on Android Tv 10.
> 
> No shortcut, app opens from settings->system apps but cannot get passed the login screen. The entire screen doesn't show properly and cuts off some of the 3 button options which cannot be clicked with remote.
> 
> You can reproduce this issue on phone by rotating to landscape. When I do this all 3 button options are completely off screen. Portrait is the only way to get passed this. Landscape works properly on all other screens.

Also, if it's not too much work can you send me a screenshot of the UI where that happens?

**Comment by DarkCrypt** [comment]
V1.4.1
✅ Shortcut works
✅ Login screen is scrollable and has access to all buttons
❌ Tv remote function(can't click)

Because of the tv remote function not being able to click, users are stuck on the login screen. Mouse is still necessary.

**Comment by BeamlakAschalew** [comment]
Yeah, I will add the functionality to scroll through buttons and movie/tv title items in the next version.

**Comment by BeamlakAschalew** [comment]
I will develop a standalone application for Android TV, we will announce the completion here and via notification.

**Comment by DarkCrypt** [comment]
> I will develop a standalone application for Android TV, we will announce the completion here and via notification.

Awesome! Definetly looking forward to this.

**Comment by Stevovo123** [comment]
This is such a great app, if it worked on Android TV it would get a lot more support

**Comment by DarkCrypt** [comment]
@BeamlakAschalew
> I will develop a standalone application for Android TV, we will announce the completion here and via notification.

This is interesting. [Flutter-movies](https://github.com/arafatamim/flutter-movies) for Android TV.

**Comment by RayzaEFC** [comment]
Wow it looks great, can't wait

**Comment by RayzaEFC** [comment]
> @BeamlakAschalew
> 
> > I will develop a standalone application for Android TV, we will announce the completion here and via notification.
> 
> This is interesting. [Flutter-movies](https://github.com/arafatamim/flutter-movies) for Android TV.

Wow look great, can't wait

**Comment by BeamlakAschalew** [comment]
@RayzaEFC the latest Android TV project mentioned isn't mine btw :(

**Comment by DarkCrypt** [comment]
@RayzaEFC 
My apologies friend. I didn't mean to cause any confusion. I mentioned the above project to BeamlakAschalew in order to help him with his Android TV version. Maybe he could reference the code, gain ideas, etc.

@BeamlakAschalew 
It would be smart to see how that project handles remote control functions.

**Comment by BeamlakAschalew** [comment]
> @RayzaEFC My apologies friend. I didn't mean to cause any confusion. I mentioned the above project to BeamlakAschalew in order to help him with his Android TV version. Maybe he could reference the code, gain ideas, etc.
> 
> @BeamlakAschalew It would be smart to see how that project handles remote control functions.

That's alright

Sure I'll check it out
